src/Raven.Server/Documents/Indexes/Index.cs

```diff
@@ -2283,6 +2287,18 @@ public bool DoIndexingWork(IndexingStatsScope stats, CancellationToken cancellat
             }
         }
 
+        private void HandleReferences(RavenTransaction tx)
+        {
+            _indexStorage.WriteReferences(CurrentIndexingScope.Current, tx);
+
+            if (_updateReferenceLoadWarning == false)
+                return;
+            
+            DocumentDatabase.NotificationCenter.Indexing.AddWarning(Name, _referenceLoadWarning);
```

Comment:
We need to add the document id here, I think

Reply:
We already have the ID. See `ReferenceId` in the following class:

```
public class LoadedReference
{
	public string ReferenceId { get; set; }

	public int NumberOfLoads { get; set; }
}
```

Reply:
You aren't showing that in the alert, however

Reply:
It's going to be available in the details of the alert. It requires changes on Studio side.

Reply:
See comment: https://issues.hibernatingrhinos.com/issue/RavenDB-16726#focus=Comments-67-90182.0-0